packages/amplify-opensearch-simulator/src/index.ts

```diff
@@ -60,10 +60,10 @@ export class OpenSearchEmulator {
     return `http://localhost:${this.port}/`;
   }
 
-  public terminate() {
+  public terminate(): Promise<unknown> {
```

Comment:
is this return value used anywhere? If not, the return type should be `Promise<void>`